test/module.test.js

```diff
@@ -201,6 +201,37 @@ describe('Test sidekick', () => {
     assert.ok(plugins.find((p) => p.id === 'foo'), 'Did not add plugin from config');
   }).timeout(IT_DEFAULT_TIMEOUT);
 
+  it('Adds badge plugin to feature container', async () => {
+    const setup = new Setup('blog');
+    nock.sidekick(setup, {
+      configJson: `{
+        "plugins": [{
+          "id": "foo",
+          "title": "Foo",
+          "isBadge": true,
+          "badgeTextColorLight": "red"
```

Comment:
To test all code branches, can you create 2 badges, one using all default colors, the other overriding all four colors?

Reply:
I think you could inject an inline `style` element with the plugin's `elements` array, where you have the freedom to specify color scheme specific colors, e.g.
```css
.plugin.badge#id {
  color: black;
  background-color: red;
}

@media (prefers-color-scheme: dark) {
  .plugin.badge#id {
    color: white;
    background-color: blue;
  }
}
```